ext/vscode/src/commands/cmdUtil.ts

```diff
@@ -92,8 +92,8 @@ export async function selectApplicationTemplate(context: IActionContext): Promis
         .withArg('--output').withArg('json')
         .build();
     const result = await execAsync(command);
-    const templates = JSON.parse(result.stdout) as { name: string, description: string, repositoryPath: string }[];
-    const choices = templates.map(t => { return { label: t.name, detail: t.description, data: t.repositoryPath } as IAzureQuickPickItem<string>; });
+    const templates = JSON.parse(result.stdout) as { name: string, description: string, path: string }[];
+    const choices = templates.map(t => { return { label: t.name, detail: t.description, data: t.path } as IAzureQuickPickItem<string>; });
```

Comment:
@karolz-ms / @bwateratmsft This ends up breaking users with latest vscode and older CLI, but maybe ok before GA? I can also add a check for presence of both fields. Let me know what we prefer.

Reply:
I'm OK with that before GA. Enforcing a new minimum version is as easy as updating this line: https://github.com/Azure/azure-dev/blob/b42c228e28c58c7afa428b893e414cfe002090eb/ext/vscode/src/utils/azureDevCli.ts#L56

Updating in VSCode is super painless since we're using the script install, literally all that users will have to do is click the "Update" button in the toast or run the "Install / Update Azure Developer CLI" command

Reply:
(see my earlier comment about keeping the `repositoryPath` property name)

Reply:
Reverted this change. Thanks for the feedback.